airgun/entities/host.py

```diff
@@ -153,6 +153,28 @@ def schedule_remote_job(self, entities_list, values, timeout=60, wait_for_result
             status_view.wait_for_result(timeout=timeout)
         return status_view.read()
 
+    def get_puppet_class_parameter_value(self, entity_name, name):
```

Comment:
1 - as we have set_puppet_class_parameter_value, it's logical to have a get_puppet_class_parameter_value
2- yes we can do host.read(), but we have to filter that table each time, we have many tests that need only that parameter value , it's logical to reduce time to read only that value here in airgun, and to reduce time and efforts in tests getting only the needed value.

Reply:
i remember we discussed that on mtg before, main point was: `read(widget_names='parameters.puppet_class_parameters')` can be used, but only to read all values of the table, and when we want to use some filtering/searching, we create separate helper for that, so that's why this method is needed.